cmd/create/cluster/cmd.go

```diff
@@ -4062,7 +4062,7 @@ func getSecurityGroups(r *rosa.Runtime, cmd *cobra.Command, isVersionCompatibleC
 			os.Exit(1)
 		}
 		*additionalSgIds = interactiveSgs.
-			GetSecurityGroupIds(r, cmd, vpcId, kind)
+			GetSecurityGroupIds(r, cmd, vpcId, kind, "")
```

Comment:
One last question before LGTM from me :slightly_smiling_face: 
During cluster create flow, if the user opts in to use additional sec. groups, I'm wondering if the list of security groups visible should be filtered to the ones that is **not owned** by any cluster.
From what I recall, the resource tag 
kubernetes.io/cluster/{cluster_id}=owned is also used when deleting / cleaning up a cluster.

Please correct me here if I'm incorrect . Trying to better understand the flow!

Reply:
At cluster creation, we don't have cluster/infra ID to check the owned tag.
If we decide to filter security groups that are owned by any cluster, checking the substring is sufficient, as you suggested.
I agree with your suggestion.
@gdbranco and I had a similar discussion, we can confirm with BU/SRE and open a ticket to track changes in the CLI and backend, to validate if a security group is owned by any cluster.

Reply:
@davidleerh created this ticket https://issues.redhat.com/browse/OCM-7427 to track filtering upon cluster creation.